sdk/eventhub/Azure.Messaging.EventHubs.Shared/tests/Processor/PartitionLoadBalancerTests.cs

```diff
@@ -0,0 +1,413 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Messaging.EventHubs.Diagnostics;
+using Azure.Messaging.EventHubs.Tests;
+using Moq;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventHubs.Processor.Tests
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="PartitionLoadBalancer" />
+    ///   class.
+    /// </summary>
+    ///
+    [TestFixture]
+    public class PartitionLoadBalancerTests
+    {
+        private const string FullyQualifiedNamespace = "fqns";
+        private const string EventHubName = "name";
+        private const string ConsumerGroup = "consumerGroup";
+
+        /// <summary>
+        ///   Verifies that partitions owned by an <see cref="PartitionLoadBalancer" /> are immediately available to be claimed by another loadbalancer
+        ///   after StopAsync is called.
+        /// </summary>
+        ///
+        [Test]
+        public async Task RelinquishOwnershipAsyncRelinquishesPartitionOwnershipOtherClientsConsiderThemClaimableImmediately()
+        {
+            using CancellationTokenSource tokenSource = new CancellationTokenSource();
```

Comment:
The token source seems useless here as well because there's nothing to trigger the cancellation. Before moving the tests we had a time constraint so the test didn't hang forever.

https://github.com/Azure/azure-sdk-for-net/blob/b78b6637efc1d673f7b2e7faa0b0f4de2a60fd73/sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientTests.cs#L1015-L1018

Did you forget to include it? Other tests seem to be missing it as well.

In case you removed it intentionally, there's no point in keeping the `tokenSource` (unless I'm missing something, which may be the case).

Reply:
Since the load balancer itself no longer has any wait time logic, it's no longer necessary to have a time constraint for these tests. Each execution of `RunLoadBalancingAsync` runs and completes almost immediately. The token is just there to satisfy the required arguments.

Reply:
You have a point. I would suggest following Jesse's recommendation of using a `CancellationToken.None` because these seem more appropriate for these situations. [You'll get the same effect if you simply use 'default' instead](https://docs.microsoft.com/en-us/dotnet/api/system.threading.cancellationtoken.none?view=netstandard-2.0#remarks).

It's just a matter of preference, so I'll leave the change at your discretion.